### ISSUE TITLE
Windows specific path finding problem

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/leanbuild#49]
```
No root directory found in C:/Users/runneradmin/AppData/Local/Temp/RtmpER1MgI/Rbuild128c55b62912/leanbuild/vignettes or its parent directories. Root criterion: contains a file "_bookdown.yml"
```
From: https://github.com/jhudsl/leanbuild/pull/47/checks?check_run_id=3610942536

I don't know what the deal is here. Tried to make sure it wasn't fixed by just a `file.path()` here and there, but it was not.

**Comment by cansavvy** [comment]
I think `rprojroot` package has some inconsistencies when it comes to windows and we use that a fair amount. https://github.com/r-lib/rprojroot/issues/36

**Comment by cansavvy** [comment]
^ I was wrong it's not rprojroot's problem. I think it has to do with the zipping or unzipping of the example files. But I don't know how to fix it. 

Why does it think it needs to find this file? I'm not sure. 

```
Quitting from lines 70-72 (getting-started.Rmd) 

Error: Error: processing vignette 'getting-started.Rmd' failed with diagnostics:

cannot open file './docs/coursera/resources/images/home/rstudio/DaSL_Course_Template_Bookdown/docs/coursera/01-intro_files/figure-html/1YmwKdIy9BeQ3EShgZhvtb3MgR8P6iDX4DfFD65W_gdQ_gcc4fbee202_0_141.png': No such file or directory

--- failed re-building 'getting-started.Rmd'



Warning in file.remove(f) :

  cannot remove file 'example-repo.zip', reason 'Permission denied'

SUMMARY: processing the following file failed:

  'getting-started.Rmd'



Error: Error: Vignette re-building failed.

Execution halted

Error: Error in proc$get_built_file() : Build process failed
Calls: <Anonymous> ... build_package -> with_envvar -> force -> <Anonymous>
Execution halted
Error: Process completed with exit code 1.
```

**Comment by cansavvy** [comment]
This should be addressed at the same time as #56 So it no longer needs to be its own issue.
